app/domain/SignedInUser.scala

```diff
@@ -16,4 +16,4 @@
 
 package domain
 
-case class SignedInUser(eori: String, allEoriHistory: Seq[EoriHistory])
+case class SignedInUser(eori: String, allEoriHistory: Seq[EoriHistory], XiEori: Option[String])
```

Comment:
Please change XiEori to either xiEori or xiEORI as it should follow lowerCamelCase naming convention for vals